### ISSUE TITLE
Missing playlists in ID

### ISSUE THREAD

**Issue by superbonaci** [ihabunek/twitch-dl#163]
1080p60 and 1440p60 (chunked) playlists are missing:

```
$ twitch-dl info 'https://www.twitch.tv/r0dn3y/video/2223719525'
Fetching video...
Fetching access token...
Fetching playlists...
Fetching chapters...

Video 2223719525
[QSV AV1 1440p] Minecraft + Sable
r0dn3y playing Minecraft
Description: *   1440p60 AV1 @ 12 Mbps (QSV)

*   1080p60 HEVC @ 8 Mbps (QSV)

*   720p60 H.264 @ 3.5 Mbps (QSV)

*   480p30 H.264 @ 1 Mbps (QSV)
Published 2024-08-14 @ 01:26:31  Length: 2 h 13 min
https://www.twitch.tv/videos/2223719525

Playlists:
720p60 https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/720p60/index-dvr.m3u8
480p https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/480p30/index-dvr.m3u8
Audio Only https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/audio_only/index-dvr.m3u8

Chapters:
00:00:00 Minecraft (01:28:27)
01:28:27 Sable (45:30)

Placeholder      Value
---------------  -------------------------------
{channel}        r0dn3y
{channel_login}  r0dn3y
{date}           2024-08-14
{datetime}       2024-08-14T01:26:31Z
{format}         mkv
{game}           Minecraft
{game_slug}      minecraft
{id}             2223719525
{time}           01:26:31Z
{title}          [QSV AV1 1440p] Minecraft Sable
{title_slug}     qsv_av1_1440p_minecraft_sable
```

**Comment by ihabunek** [comment]
Was a bit tricky, but done. Released in 2.5.0. Please test.

**Comment by superbonaci** [comment]
Yes now works, thanks.

<details>
  <summary>info log</summary>
/opt/homebrew/bin/twitch-dl info 'https://www.twitch.tv/r0dn3y/video/2223719525'
Fetching video...
Fetching access token...
Fetching playlists...
Fetching chapters...

Video 2223719525
[QSV AV1 1440p] Minecraft + Sable
r0dn3y playing Minecraft
Description: *   1440p60 AV1 @ 12 Mbps (QSV)

*   1080p60 HEVC @ 8 Mbps (QSV)

*   720p60 H.264 @ 3.5 Mbps (QSV)

*   480p30 H.264 @ 1 Mbps (QSV)
Published 2024-08-14 @ 01:26:31  Length: 2 h 13 min
https://www.twitch.tv/videos/2223719525

Playlists:
1440p60 https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/chunked/index-dvr.m3u8
720p60 https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/720p60/index-dvr.m3u8
480p https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/480p30/index-dvr.m3u8
1080p60 https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/1080p60/index-dvr.m3u8
Audio Only https://dgeft87wbj63p.cloudfront.net/09b88c14ef8300ea9fc4_r0dn3y_42737372968_1723598775/audio_only/index-dvr.m3u8

Chapters:
00:00:00 Minecraft (01:28:27)
01:28:27 Sable (45:30)

Placeholder      Value
---------------  -------------------------------
{channel}        r0dn3y
{channel_login}  r0dn3y
{date}           2024-08-14
{datetime}       2024-08-14T01:26:31Z
{format}         mkv
{game}           Minecraft
{game_slug}      minecraft
{id}             2223719525
{time}           01:26:31Z
{title}          [QSV AV1 1440p] Minecraft Sable
{title_slug}     qsv_av1_1440p_minecraft_sable
</details>
